integration-tests-ex/cases/src/test/java/org/apache/druid/testsEx/msq/ITMultiStageQuery.java

```diff
@@ -234,11 +236,15 @@ public void testExport() throws Exception
 
     msqHelper.pollTaskIdForSuccess(resultTaskStatus.getTaskId());
 
-    Map<String, MSQTaskReport> statusReport = msqHelper.fetchStatusReports(resultTaskStatus.getTaskId());
-    MSQTaskReport taskReport = statusReport.get(MSQTaskReport.REPORT_KEY);
+    Map<String, TaskReport> statusReport = msqHelper.fetchStatusReports(resultTaskStatus.getTaskId());
+
+    MSQTaskReport taskReport = (MSQTaskReport) statusReport.get(MSQTaskReport.REPORT_KEY);
     if (taskReport == null) {
       throw new ISE("Unable to fetch the status report for the task [%]", resultTaskStatus.getTaskId());
     }
+    TaskContextReport taskContextReport = (TaskContextReport) statusReport.get(TaskContextReport.REPORT_KEY);
+    Assert.assertFalse(taskContextReport.getPayload().isEmpty());
```

Comment:
Can we have some more meaningful assertions? If not relevant here, then we can just skip this assertion too.

Reply:
This is to assure TaskContextReport is generated for the task. Since the payload is task context which is hard to know the exact value for comparison so here it just check whether the field is populated.